portfolio/src/main/webapp/calendar.js

```diff
@@ -0,0 +1,50 @@
+class CalendarEvent{
```

Comment:
1. I think there should be a space before {. Can we use some tool that can automatically detect these style issues?
2. Can we have some comment about this class?
https://google.github.io/styleguide/jsguide.html#formatting-comments

Reply:
Updated.